src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/ServiceAccountProviderHelper.scala

```diff
@@ -82,4 +82,6 @@ class ServiceAccountProviderHelper(wrappedServiceAccountProvider: ServiceAccount
       wrappedServiceAccountProvider.listUsersStagingBucketReaders(userEmail)
     }
   }
+
+  override def getAccessToken(userEmail: WorkbenchEmail, googleProject: GoogleProject): String = wrappedServiceAccountProvider.getAccessToken(userEmail, googleProject)
```

Comment:
I think we want to wrap this in a `safeCall { ... }` similar to the other methods in this class.